Api/src/controllers/partyController.js

```diff
@@ -0,0 +1,85 @@
+// Leaving this here for future reference to create dynamic id 
+// import {
+//   v4 as uuidv4
+// } from 'uuid';
+
+import fs from 'fs';
+import path from 'path';
+const __dirname = path.resolve();
+
+import data from "../data.json" assert { type: "json" };;
+
+let parties = [{
+  id: "",
+  name: "",
+  hqAddress: "",
+  logoUrl: ""
+}];
+
+export const getParties = (req, res) => {
+  res.send(data.Parties);
+};
+
+export const createParty = (req, res) => {
+  let party = req.body;
+
+  data.Parties.push(party);
```

Comment:
This is the correct way to do it, as oppose to the `fs` you're using in some places to save data.

Reply:
I got the one now.